spec/routes/web/load_balancer_spec.rb

```diff
@@ -223,7 +223,7 @@
         expect(page).to have_flash_notice("VM is attached to the load balancer")
         expect(lb.vms.count).to eq(1)
 
-        expect(Config).to receive(:load_balancer_service_hostname).and_return("lb.ubicloud.com").twice
+        expect(Config).to receive(:load_balancer_service_hostname).and_return("lb.ubicloud.com").thrice
```

Comment:
Usually in a case like this I do `.at_least(:once)` unless I feel strongly about asserting the number of method calls...true extremely rarely. That reduces churn and annoyance over time.
```suggestion
        expect(Config).to receive(:load_balancer_service_hostname).and_return("lb.ubicloud.com").at_least(:once)
```